jvcl/run/JvStrToHtml.pas

```diff
@@ -10,8 +10,8 @@
 
 The Original Code is: JvStrToHtml.PAS, released on 2001-02-28.
 
-The Initial Developer of the Original Code is S�bastien Buysse [sbuysse att buypin dott com]
```

Comment:
Made the changes directly in browser on GitHub and I didn't touch those two lines. I have no idea how to remove them from the pull request, though :(